ocl/load-concepts-to-ocl.sh

```diff
@@ -189,11 +198,33 @@ create_hl7_medication_dispense_status_reason_source_in_ocl() {
       $OCL_API_URL/orgs/PIH/sources/
 }
 
+create_pih_malawi_source_in_ocl() {
+  curl --silent \
+      -H "Authorization: Token $OCL_API_TOKEN" \
+      -H "Accept: application/json" \
+      -H "Content-Type: application/json" \
+      --request POST \
+      --data '{"id":"PIH-Malawi","short_code":"PIHMalawi","name":"PIH Malawi","description":"(2015 or older) Partners In Health Malawi concept dictionary","source_type":"External","default_locale":"en","supported_locales":"en"}' \
+      $OCL_API_URL/orgs/PIH/sources/
+}
+
+create_liberia_moh_source_in_ocl() {
+  curl --silent \
+      -H "Authorization: Token $OCL_API_TOKEN" \
+      -H "Accept: application/json" \
+      -H "Content-Type: application/json" \
+      --request POST \
+      --data '{"id":"LiberiaMOH","short_code":"LiberiaMOH","name":"Liberia MoH","full_name":"LiberiaMOH","description":"Liberia Ministry of Health disease codes","source_type":"External","default_locale":"en","supported_locales":"en"}' \
+      $OCL_API_URL/orgs/PIH/sources/
+}
```

Comment:
In an ideal world, these wouldn't be needed, but I guess they were for the import?  We should be able to have mappings  in OCL without corresponding sources managed in OCL